### ISSUE TITLE
Feature request - delete snapshots older than x

### ISSUE THREAD

**Issue by iron9** [mrlt8/docker-wyze-bridge#558]
Snapshots can be misleading if camera disconnects and an old snapshot remains.

It would be great if snapshot is deleted after x attempts to pull new snapshot or after snapshot is x amount of time old.

Thank you!

**Comment by mrlt8** [comment]
Is this with the `/img/camera-nickname.jpg` endpoint? You need to use `/snapshot/camera-nickname.jpg` to pull the latest snapshot which should redirect to a not found placeholder if it fails.

**Comment by iron9** [comment]
The former. I am running the bridge as a Home Assistant add-on, and snapshots are generated every 15 seconds (RTSP15). I use the still_image_url: http://127.0.0.1:8123/local/cam-name.jpg with the generic camera platform.
I setup the add-on before the WebUI and on-demand snapshots were implemented. I will give that a try, thanks!

**Comment by mrlt8** [comment]
Yeah, sorry about that, the RTSP snapshots are a little broken right now. 

Eventually want to move to on-demand as the default with the snapshot interval as an option.
